src/CMakeLists.txt

```diff
@@ -317,7 +318,9 @@ ELSE()
 	MESSAGE(FATAL_ERROR "Unsupported architecture (CMAKE_SIZEOF_VOID_P is unset)")
 ENDIF()
 
-CONFIGURE_FILE(features.h.in git2/sys/features.h)
+# Generate feature headers
+CONFIGURE_FILE(features.h.in ${libgit2_BINARY_DIR}/include/git2/sys/features.h)
```

Comment:
`features.h` is actually not installed at present, this is something that's only built and used locally.  Let's keep it private API without a compelling reason to promote it.

Reply:
Fair enough, makes sense :+1: